common/utils.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+def camel2snake(text):
```

Comment:
Not really. There are some frameworks which has utility functions for that but I didn't found a lib in Python to do this. It is quite a simple function and I think we can manage to support it :)